### PR TITLE
Solve issue - #45 Solving issues #45 - Adding a functionality that lets the user know when they copy an icon

### PR DESCRIPTION
**Pull request by shrinishant**
#45 

I've used react-toastify library to implement this feature. All the changes have been made only into HowToPanel.js file and package.json (to install the library)

### CODE REVIEW

**Comment by vinayaksh42** [comment]
I am unable to check your PR as `react-toastify` has not been added as a dependency.

Error message: `Module not found: Can't resolve 'react-toastify' in 'C:\Users\leega\Desktop\EOS-Mentor\eos-icons-landing\src'`

Can you add `react-toastify` to this PR?

File: src/components/HowToPanel.js [hunk 1]
```diff
@@ -94,6 +96,16 @@ const HowToPanel = (props) => {
                   onClick={() => {
                     document.getElementById('copy-code').select()
                     document.execCommand('copy')
+
+                    toast.info('Copied !!!', {
```
Comment (vinayaksh42) [inline src/components/HowToPanel.js:100]: @shrinishant I think we should change the message to `Copied!` or `Copied to clipboard`.

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]

**Comment by sarthakkundra** [comment]
Hey @vinayaksh42. Shouldn't we create an in-house reusable toast component instead of using a library like `react-toastify`. It's ~19 KB minified which is quite huge. And seeing the usecase here I think an in-house component won't be a big task to build. Let me know what you think about that?
CC :- @shrinishant

**Comment by vinayaksh42** [comment]
> Hey @vinayaksh42. Shouldn't we create an in-house reusable toast component instead of using a library like `react-toastify`. It's ~19 KB minified which is quite huge. And seeing the usecase here I think an in-house component won't be a big task to build. Let me know what you think about that? CC :- @shrinishant

Great suggestion @sarthakkundra, I agree that an in-house toast component will be more lightweight and be a lot more customizable. @shrinishant I think we should proceed with this suggestion and try to create a toast component, let me know if you face any issue while working on it.

**Comment by shrinishant** [comment]
> > Hey @vinayaksh42. Shouldn't we create an in-house reusable toast component instead of using a library like `react-toastify`. It's ~19 KB minified which is quite huge. And seeing the usecase here I think an in-house component won't be a big task to build. Let me know what you think about that? CC :- @shrinishant
> 
> Great suggestion @sarthakkundra, I agree that an in-house toast component will be more lightweight and be a lot more customizable. @shrinishant I think we should proceed with this suggestion and try to create a toast component, let me know if you face any issue while working on it.

Sure sir, that would be great but the same "react-toastify" library has been used in the user-story project recently, the link for the pr is [here](https://github.com/EOS-uiux-Solutions/user-story/pull/140). I think we should maintain consistency in both the projects. Let me know what would be the best to proceed with.

**Comment by vinayaksh42** [comment]
> It's a separate project so I think that shouldn't affect the decision here but @vinayaksh42 can take the final call.

I think it's best to maintain some level of consistency between user story and eos icons landing.

**Comment by cyntss** [comment]
Very good decision @vinayaksh42 . I support consistency across projects too. It reduces maintenance overhead and knowledge is transferable.

**Pull request closed without merging**